pkg/libcoap/Makefile.dep

```diff
@@ -1,4 +1,5 @@
 ifneq (,$(filter libcoap,$(USEPKG)))
   USEMODULE += posix_sockets
+  USEMODULE += posix
```

Comment:
`posix_socket` should pull in `posix` as a dependency, shouldn't it? It would make sense at least.

Reply:
Yes, but if libcoap directly depends on posix headers, it should declare that dependency instead of relying on posix_sockets to pull in the headers.

Reply:
Right!